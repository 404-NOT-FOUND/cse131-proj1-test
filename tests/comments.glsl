/* hey */
/* hey
hey
hey
*/
// this is a comment.
// !@#$%%^^&*
this is not
