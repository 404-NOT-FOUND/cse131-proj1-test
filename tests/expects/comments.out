this         line 8 cols 1-4 is T_Identifier 
is           line 8 cols 6-7 is T_Identifier 
not          line 8 cols 9-11 is T_Identifier 
